### ISSUE TITLE
Blocks "More Options" menu: arrow navigation doesn't work on all the menu items

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#5245]
When opening the blocks "More Options" menu, it is possible to use the arrow keys to navigate through the items in the menu.

However, the content changes depending on the context and it's made of more than one component. Seems arrow navigation works only on the first items in the menu, I guess they're the ones from the "main" component, while it doesn't work at all on the additional items:

<img width="498" alt="screen shot 2018-02-25 at 11 25 46" src="https://user-images.githubusercontent.com/1682452/36642064-fc66929a-1a39-11e8-8274-04627392169e.png">

Easier way to reproduce:
- go in Add New post
- click the ellipsis button in the default block
- use the arrow keys to navigate the menu items
- arrows navigation works only on the first 4 items
